app/admin/users.rb

```diff
@@ -112,12 +112,18 @@ def update_resource(object, attributes)
       f.input :vaccination
       f.input :comment
       f.input :status_id, as: :select, collection: UserStatus.all.pluck(:status, :id)
-      f.input :coordinator_id, as: :select, collection: AdminUser.all.pluck(:email, :id)
-      f.input :accommodation_pref, as: :select, collection: AccommodationType.all.pluck(:name, :id)
+      f.input :coordinator_id, as: :select, collection: AdminUser.all.pluck(:name, :id)
+      f.input(
+        :accommodation_pref,
+        as: :select,
+        collection: AccommodationType.all.pluck(:name, :id),
+        selected: AccommodationType.all.pluck(:name, :id).first
+      )
       f.input(
         :transport_id,
         as: :select,
-        collection: Transport.to_select_collection
+        collection: Transport.to_select_collection,
+        selected: Transport.to_select_collection.second
```

Comment:
on edit page selected should use value picked during creation

Reply:
Done
Addressed here https://github.com/vova/war_shelter/pull/68/commits/4e4c286a607a201f83ff67501e343734d2ebb10e